src/eachrow.jl

```diff
@@ -104,29 +104,32 @@ for row in eachrow(copy(df))
 end
 ```
 
+See [`@eachrow!`](@ref) which employs the same syntax but allocates
+a fresh data frame.
```

Comment:
This is backwards: `@eachrow!` is in-place. Also I'd rather mention this a bit below, when explaining that columns are not shared. Same for `@eachrow!`.